### DISCUSSION TITLE
Obrigado FrontendBr

### DISCUSSION THREAD

**Discussion by napalmdev** [frontendbr/forum#204] [Discussão]
Fala galera, queria antes de mais nada expressar minha imensa gratidão a esta iniciativa e a todos que contribuem com ela,  e deixar esta issue pra quem quiser deixar seu depoimento de agradecimento, assim como o meu aqui \o/.

Desde a abertura deste Fórum, eu consumi muita informação boa em issues incríveis, salvei links valiosos para meus estudos de Inglês, vi relatos muitos interessantes sobre quem trabalha na gringa, e quem trabalha em Home Office, mudei de opinião e ponto de vista vááááárias vezes, conheci uma porrada de tecnologias novas e o mais incrível de tudo, arrumei um trampo em Home Office por AQUI graças às issues de Vagas, cara, que sensacional. Eu espero e quero ainda retribuir tudo isso, mas por enquanto sigo estudando e seguindo cada issue e cada comentário aqui postado, vlw galera, vcs são FODA.

:smile:

**Comment by fdaciuk** [comment]
Que legal cara! Sucesso no novo trampo! Essa troca de conhecimento da galera aqui está sendo realmente sensacional :dancer: :dancer: :sparkles:

**Comment by napalmdev** [comment]
Vlw cara, realmente, eu já perdi a conta de quantas Issues foda tem aqui

**Comment by diogomoretti** [comment]
Sucesso nessa empreitada @napalmdev!

**Comment by vitor-mendrone-deel** [comment]
Sem palavras, @napalmdev, era esse nosso objetivo desde o inicio! :heart: 

@LFeh @eduardojmatos @vitorbritto @willianjusten, @diogomoretti, @fdaciuk  acho que o objetivo está sendo atingido e com muito sucesso!

**Comment by eduardojmatos** [comment]
Poxa, que coisa maravilhosa cara! Sucesso pra vc nesse novo trampo!

Nós é que agradecemos à essa comunidade tão poderosa que tem usado o fórum e trocado informações! Somos só a fagulhazinha pra acender isso tudo!

@Mendrone achievement reached!!

**Comment by cmpereirasi** [comment]
Maravilha do isssue. A comunidade tá 💯 ... 
@napalmdev sucesso na sua novo jornada. Abs

**Comment by napalmdev** [comment]
Vlw galera, brigadão mesmo \o/

**Comment by felipefialho** [comment]
Nossa cara, se desse pra fixar issues aqui no Github faria isso agora!

SENSACIONAL seu depoimento, especialmente quando diz que conseguiu um trabalho HOME OFFICE POR AQUI. Ou seja, aprendeu um monte de coisa em discussões FODAS, conheceu uma galera que sabe MUITO e ainda vai poder colocar tudo isso em prática em um emprego novo.

Sério, nessas horas a gente vê que que um pouco de audácia sempre faz bem (recebi alguns bombardeios quando surgiu essa ideia de usar o Github como fórum!). 

Fico muito feliz quando pessoas dão depoimentos nesse sentido (já tiveram outras, inclusive durante o Front in Sampa no último sábado). Valeu mesmo e sucesso! :smile:

**Comment by davidsonfellipe** [comment]
@napalmdev Parabéns! Muito sucesso no trampo novo!!!

**Comment by willianjusten** [comment]
Aeeee, fico muito feliz que de uma conversa no Twitter e uma galera com vontade, conseguimos fazer as coisas acontecerem e ajudar outras pessoas!

Sucesso no seu trampo novo! E que você possa dividir bastante coisa aqui também=)

**Comment by napalmdev** [comment]
Vlw feras, com certeza ainda vou retribuir o favor \o/

**Comment by vitorbritto** [comment]
@napalmdev Parabéns! Sucesso meu caro! =]

**Comment by andremachadodev** [comment]
Parabéns pelo trampo e parabéns pela escolha do nick!
